### ISSUE TITLE
The Scaffold module is unstable

### ISSUE THREAD

**Issue by Nippaku-Zanmu** [MeteorDevelopment/meteor-client#4762]
### Describe the bug

When you have the speed status effect and enable the Scaffold module the Scaffold module will become unstable as it will not automatically place blocks in time and this will cause the player to fall as shown in the video

https://github.com/user-attachments/assets/36d67cea-f844-45a7-bb65-62b75252ff92

### Steps to reproduce

1. Get speed status effect
2. Enable the Scaffold module
3. Hold the block and move

### Meteor Version

0.5.8

### Minecraft Version

1.21

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by Arson-Arson** [comment]
Turning on either air place or place ahead in the module settings should fix the problem.

**Comment by Nippaku-Zanmu** [comment]
Enabling the AirPlace option will result in the placement of additional blocks and will be detected by anti-cheat on most servers


> Turning on either air place or place ahead in the module settings should fix the problem.

**Comment by Arson-Arson** [comment]
Fair enough, but place ahead should still work; or some combination of "place per tick" or one of the other settings. And wouldn't it also depend on ping in some cases?
